app/frontend/packs/further_conditions/updateAddButtonVisibility.js

```diff
@@ -0,0 +1,15 @@
+import { ADD_BUTTON_CSS_CLASS, ITEM_CSS_CLASS } from './constants'
+
+const MAX_FURTHER_CONDITIONS = 18
```

Comment:
Not entirely happy with this.

Maybe we could render this hidden in the form html somewhere and read from it in the JS?

Reply:
Yea maybe in a data hash in hidden html, if we were using `js.erb` partials would have been easier but we aren't here so I would go with the first option